Formula/pyside.rb

```diff
@@ -19,46 +19,60 @@ class Pyside < Formula
 
   depends_on "cmake" => :build
   depends_on "ninja" => :build
+  depends_on xcode: :build
   depends_on "llvm"
   depends_on "python@3.9"
   depends_on "qt"
 
+  uses_from_macos "libxml2"
+  uses_from_macos "libxslt"
+
   def install
-    xy = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
+    inreplace "sources/pyside6/cmake/Macros/PySideModules.cmake",
+              "${shiboken_include_dirs}",
+              "${shiboken_include_dirs}:#{Formula["qt"].opt_include}"
```

Comment:
This issue has been reported to upstream.

Reply:
Can you link to the upstream report?